### ISSUE TITLE
To reproduce the default performance of the LTX VAE, apart from changing the learning rate to 2e-4, how many training steps are required for the two stages respectively?

### ISSUE THREAD

**Issue by k15201363625** [hustvl/Turbo-VAED#8]
Good job!

Currently, we try to training the default LTX VAE 40M, 70K steps for stage1, and 40K steps for stage2.

                       LPIPS, PSNR, SSIM
After Stage1, 0.0751 28.768 0.8696 
After Stage2, 0.0694 29.127 0.8806
And the training loss has decrease slowly, but the performance gap between the trained model and teacher LTX VAE is large. 

Are there any other modifications in the training script that I might have missed?
Thanks.

**Comment by k15201363625** [comment]
For training dataset, I use 20k videos randomly sampled from VidGen dataset, and the batch_size is 64. 
Furthermore, for HunyuanVideo and Cogvideo VAE in paper, what is the approximate computational resources required for training, because of 10x slower training speed than compressed LTX VAE.

**Comment by ZouYa99** [comment]
Thank you for your attention and for attempting to reproduce the results.

Firstly, regarding your first question, I'd like to know which test set your reported reproduction metrics are based on. This information will help me better understand your question. Additionally, you could try adjusting the batch size. In my experiments, an effective batch size of 16 was used (an actual batch size of 2 with 8 gradient accumulation steps).

Secondly, regarding computational resources for HunyuanVideo and Cogvideo VAE:
Actually, while these two might run slower, they converge faster during the distillation process. This is because their compression ratio is smaller, meaning the latents contain more information. Therefore, training these VAEs doesn't necessarily require more training resources than an LTX VAE.

**Comment by sdiazlor** [comment]
Hey @k15201363625, maybe this new VAED for LTX 2.3 and focused on faster decoding is of interest to your research: https://huggingface.co/PrunaAI/PrunaVAED. WDYT?
